src/seedu/addressbook/commands/Command.java

```diff
@@ -26,6 +27,10 @@ public Command(int targetIndex) {
     protected Command() {
     }
 
+    public boolean isMutating() {
```

Comment:
Having a default implementation for `isMutating()` in the parent class is good as it reduces the amount of code. However, using an arbitrary default return value (like `true`) would reduce code understandability - as its subclasses implementations are returning `false`. Instead, you could declare `isMutating()` as an abstract method with no method body.